Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: default
+
+default: pull-request-ci
+
+pipeline-build-amd64:
+	cargo build --release --features bpf \
+		&& ./package.sh amd64/rezolus.tar.gz target/release/rezolus \
+
+pipeline-build-arm64v8:
+	/sbin/bpftool btf dump file /sys/kernel/btf/vmlinux format c > src/common/bpf/vmlinux.h \
```

Comment:
you don't need to do this `bpftool` thing for amd64?

Reply:
According to Brian I don't think so, but if it goes wrong then I will come back and fix it! Thanks.